src/js/components/Modals.js

```diff
@@ -146,6 +162,20 @@ var Modals = React.createClass({
     return <CliInstallModal open={showModal} {...options} />;
   },
 
+  getClusterLinkingModal(showModal) {
+    var onClose = function() {
```

Comment:
This exact same `onClose` function is re-defined 4 times in closures in this file. Maybe take this as an oppertunity to refactor into a paramaterized:

```
onClose(modal) {

Reply:
Looks like we have refactoring Modals already in scope - lets not waste time doing it quick now, if it will be done correctly in the future :+1: